src/buttons/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -165,37 +147,16 @@ const CreateScenarioDialog = ({
     handleCloseDialog();
   };
 
-  const handleChangeScenarioMaster = (event) => setMaster(event.target.checked);
-
-  const handleChangeDataset = (newDataset) => setDatasetFieldValues(newDataset);
-
-  const handleChangeParentScenario = (newParentScenario) => setParentScenarioFieldValues(newParentScenario);
-
-  const handleScenarioTypeChange = (newScenarioType) => setScenarioTypeFieldValues(newScenarioType);
-
-  let createScenarioDisabled = true;
-  if (
-    isDialogDataValid(
-      scenarioNameFieldValues,
-      isMaster,
-      scenarioTypeFieldValues,
-      parentScenarioFieldValues,
-      datasetFieldValues
-    )
-  ) {
-    createScenarioDisabled = false;
-  }
-
-  const resetToDefaultData = () => {
-    setParentScenarioFieldValues(defaultParentScenario.current);
-    setDatasetFieldValues({});
-    setMaster(false);
-    setScenarioNameFieldValues(scenarioNameInitialState);
-    setScenarioTypeFieldValues(defaultScenarioType.current);
-  };
+  const createScenarioDisabled = !isDialogDataValid(
+    scenarioName,
+    scenarioNameFieldError,
+    isMaster,
+    selectedRunTemplate,
+    parentScenarioFieldValues,
+    datasetFieldValues
+  );
 
   const handleCloseDialog = () => {
-    resetToDefaultData();
     closeDialog();
   };
```

Comment:
Do we need to keep `handleCloseDialog()` if it's just calling another function now ? :slightly_smiling_face:

Reply:
Nice catch, I removed `handleCloseDialog` :smile: